src/packages/@app/models/guest-user.model.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  toTypedRxJsonSchema,
+  type ExtractDocumentTypeFromTypedRxJsonSchema,
+  type RxJsonSchema,
+} from "rxdb";
+
+export const guestUserSchemaLiteral = {
+  title: "Guest User Details",
+  primaryKey: "id",
+  type: "object",
+  version: 0,
+  properties: {
+    id: {
+      type: "string",
+      maxLength: 100,
+    },
+    name: {
+      type: "string",
+    },
+    isGuestUser: {
+      type: "boolean",
+    },
+    isBannerActive: {
+      type: "boolean",
+    },
+  },
+} as const;
```

Comment:
as the model suggests. it is a guest user, why is there property called `isGuestUser`

Reply:
we are also storing isBannerActive and can also need to store other things in future that's why isGuestUser name particular variable is their although we can just check if guest user details exist can work or not it would have also worked but to make it more convinent we made a separate variable for that.

Reply:
still not sure what isGuestUser does? @gc-codes